### ISSUE TITLE
only/exclude config support

### ISSUE THREAD

**Issue by G07cha** [G07cha/MineflayerArmorManager#1]

**Comment by Frettarix** [comment]
Was the config functionality implemented yet? I tried something similar by checking for armor values ['iron','diamond'] and then pulling those item_ids to allow them to be worn.

**Comment by G07cha** [comment]
Nope, I'm currently moving codebase to typescript so there are quite a bit of major changes coming up soon. I'll try to look at this after the migration.

**Comment by Frettarix** [comment]
I'm very curious of the result, I'm still using javascript together with Node to get my bots in, which also gives me the ability to create an easy Web UI, what are cons and pros for typescript?

**Comment by G07cha** [comment]
It's done already by the way, feel free to explore. Typescript is a subset of javascript which basically adds strict types to the language,  it's a two edged sword since it allows to create code less prone to errors by sacrificing speed of development because you need to write same code as in javascript but also add types and ensure they are being followed. You can still easily use this module in your javascript project, I just personally find typescript better in a long run 🙂

**Comment by Frettarix** [comment]
Thanks for the heads up buddy, will look into Typescript! ;)
